### PR TITLE
incorporate latest boilerplate changes

### PR DESCRIPTION
**Pull request by honnix**
This reflects changes done in https://github.com/lyft/boilerplate/pull/4

### CODE REVIEW

File: boilerplate/lyft/golang_test_targets/download_tooling.sh [hunk 1]
```diff
@@ -22,7 +22,6 @@ go_install_tool () {
 # List of tools to go get
 # In the format of "<cli>:<package>" or ":<package>" if no cli
 tools=(
-  "pflags:github.com/lyft/flytestdlib/cli/pflags"
```
Comment (honnix) [inline boilerplate/lyft/golang_test_targets/download_tooling.sh]: This doesn't work. I guess due to the funky dependency of itself.

File: Makefile [hunk 2]
```diff
@@ -20,6 +20,7 @@ compile:
 	go build -o pflags ./cli/pflags/main.go && mv ./pflags ./bin
 
 gen-config:
+	# Note that installing it via download_tooling.sh doesn't work
```
Comment (EngHabu) [inline Makefile]: Why is that?

**Comment by honnix** [comment]
Not sure `lyft/golangci_file` is not there. I removed it manually after `make update_boilerplate`. If we need it that can be a separated PR.

**Comment by honnix** [comment]
@wild-endeavor PTAL

**Pull request merged**